.github/workflows/build-scan-push.yaml

```diff
@@ -129,3 +134,18 @@ jobs:
             ${{ steps.docker_meta.outputs.labels }}
             ```
           edit-mode: replace
+
+      # Compare the image built in the pull request with the one in production
+      - name: Docker Scout
+        id: docker-scout
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: docker/scout-action@v1
+        with:
+          command: compare
+          image: ${{ steps.docker_meta.outputs.tags }}
+          to: docker/${{ github.event.repository.name }}:latest
```

Comment:
Wouldn't you want to compare it against the highest semver that is published in the Marketplace instead of `latest`?

Reply:
Do you know how to do that? That was my whish but it is not built-in and then I would need to "manually" retrieve the latest semver tag